server/src/main/java/org/apache/druid/client/selector/QueryableDruidServer.java

```diff
@@ -19,38 +19,16 @@
 
 package org.apache.druid.client.selector;
 
-import org.apache.druid.client.DirectDruidClient;
 import org.apache.druid.client.DruidServer;
+import org.apache.druid.query.QueryRunner;
 
 /**
+ * Interface to represent a single server which can participate in query processing. The main implementation is
+ * {@link RemoteDruidServer}.
  */
-public class QueryableDruidServer
+public interface QueryableDruidServer<T extends QueryRunner>
 {
-  private final DruidServer server;
-  private final DirectDruidClient client;
+  DruidServer getServer();
```

Comment:
This interface existed before I changed and I don't want to change it in this PR because the refactoring was just to make it easy to write the benchmark code.